packages/website/src/components/playground/OptionsSelector.tsx

```diff
@@ -0,0 +1,147 @@
+import type { ConfigModel } from './types';
+
+interface OptionsSelectorParams {
+  readonly config: ConfigModel;
+  readonly enableScrolling: boolean;
+  readonly setEnableScrolling: (checked: boolean) => void;
+  readonly setConfig: (cfg: Partial<ConfigModel>) => void;
+}
+
+function OptionsSelectorContent({
+  config,
+  setConfig,
+  enableScrolling,
+  setEnableScrolling,
+}: OptionsSelectorParams): JSX.Element {
+  const windowSize = useWindowSize();
+
+  const [copyLink, copyLinkToClipboard] = useClipboard(() =>
+    document.location.toString(),
+  );
+  const [copyMarkdown, copyMarkdownToClipboard] = useClipboard(() =>
+    createMarkdown(config),
+  );
+
+  const openIssue = useCallback((): void => {
+    const params = createMarkdownParams(config);
+
+    window
+      .open(
+        `https://github.com/typescript-eslint/typescript-eslint/issues/new?${params}`,
+        '_blank',
+      )
+      ?.focus();
+  }, [config]);
+
+  return (
+    <>
+      <Expander label="Info">
+        <InputLabel name="TypeScript">
+          <Dropdown
+            name="ts-version"
+            options={tsVersions}
+            value={config.ts}
+            onChange={(ts): void => setConfig({ ts })}
+          />
+        </InputLabel>
+        <InputLabel name="Eslint">{process.env.ESLINT_VERSION}</InputLabel>
+        <InputLabel name="TSEslint">{process.env.TS_ESLINT_VERSION}</InputLabel>
+      </Expander>
+      <Expander label="Options">
+        <InputLabel name="Source type">
+          <Dropdown
+            name="sourceType"
+            value={config.sourceType ?? 'module'}
+            onChange={(sourceType): void => setConfig({ sourceType })}
+            options={['script', 'module']}
+          />
+        </InputLabel>
+        <InputLabel name="File type">
+          <Dropdown
+            name="fileType"
+            value={config.fileType ?? 'ts'}
+            onChange={(fileType): void => setConfig({ fileType })}
+            options={fileTypes}
+          />
+        </InputLabel>
+        <InputLabel name="Auto scroll">
+          <Checkbox
+            name="enableScrolling"
+            value=""
+            checked={enableScrolling}
+            onChange={setEnableScrolling}
+          />
+        </InputLabel>
+      </Expander>
+      <Expander label="Actions">
+        <ActionLabel name="Copy link" onClick={copyLinkToClipboard}>
+          <Tooltip open={copyLink} text="Copied">
+            <CopyIcon width="13.5" height="13.5" />
+          </Tooltip>
+        </ActionLabel>
+        <ActionLabel name="Copy Markdown" onClick={copyMarkdownToClipboard}>
+          <Tooltip open={copyMarkdown} text="Copied">
+            <CopyIcon width="13.5" height="13.5" />
+          </Tooltip>
+        </ActionLabel>
+        <ActionLabel name="Report as Issue" onClick={openIssue}>
+          <IconExternalLink width="13.5" height="13.5" />
+        </ActionLabel>
+      </Expander>
+      {windowSize !== 'mobile' && (
```

Comment:
Restricting some information on mobile viewports is an accessibility violation - some viewers have to zoom in a bunch and/or have unexpected viewport resolutions. And having dev tools open side-by-side often makes the playground snap to mobile...

```suggestion
```

Reply:
this block will be removed before we merge it, as i left it to ease debugging during review of changes, in mobile it make no sense to actually display those